bundles/org.openhab.binding.volumio/src/main/resources/OH-INF/i18n/volumio_de_DE.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
This file should be removed because all translations should be generated using Crowdin.

Reply:
resolved